### PR TITLE
Ad landing page template

### PR DESCRIPTION
**Pull request by homeofmatt**
TODO
- [x] Build html for ad landing page widgets
- [x] Import sample media files
- [x] Write styles to make html match design
- [x] Write necessary javascript for messaging widgets

### CODE REVIEW

**Comment by homeofmatt** [comment]
Currently functioning to display progress, NOT READY FOR REVIEW.

File: app/views/widgets/ad_landing_page/_masthead.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      </article>
+    </div> 
+  </div>
+
+  <div class="masthead-content">
+    <section class="masthead-content-panel">
+      <h2>Take the first step to advance your career.</h2>
+
+      <div class="calls-to-action">
+        <div id="form_a4517720-e7d7-4e0e-bafd-2c1045918e0d">Loading...</div><script>/*<![CDATA[*/var script = document.createElement('script'); script.async = 1; script.src = 'https://go.chapman.edu/register/?id=a4517720-e7d7-4e0e-bafd-2c1045918e0d&output=embed&div=form_a4517720-e7d7-4e0e-bafd-2c1045918e0d' + ((location.search.length > 1) ? '&' + location.search.substring(1) : ''); var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(script, s);/*]]>*/</script>
```
Comment (tatwell) [inline app/views/widgets/ad_landing_page/_masthead.html]: Is this the form code? Let's add comment explaining what it does.

Break js into multiple lines if possible.

Also, why CDATA wrapper for js?

File: app/views/widgets/ad_landing_page/_messaging_with_image.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div class="ad-landing-messaging-widget ad-landing-messaging-widget__image ad-landing-messaging-widget__text-left">
+  <div class="outer-container">
+    <div class="text-column">
+      <div class="section-message">
+        <h3 class="title">Read what one of our graduates has to say:</h3>
+        <div class="body editableContent">
+          <p>
+            <img class="media" src="http://www.critterbabies.com/wp-content/gallery/kittens/Cute-Kitten-kittens-16122946-1280-800.jpg"/>
+            "I came to Chapman for reason: QUALITY. Quality in education, quality in professors, quality in career services, and quality in my fellow classmates. Chapman is constantly striving for greatness, and they have given me the resources to achieve the same throughout my hourney. Because of the intimate setting Chapman provides, students are able to take full advantage of the resources and services they offer and, most importantly, get to know the faculty, staff, and fellow classmates on a more casual level."<br /><br /><strong>Jackie Becerra, MBA '15</strong><br />Nike, Inc. Intern
```
Comment (tatwell) [inline app/views/widgets/ad_landing_page/_messaging_with_image.html]: Going forward, let's try to keep lines in templates under 100 char limit.

File: lib/tasks/test.rake [hunk 3]
```diff
@@ -0,0 +1,18 @@
+require 'rubocop/rake_task'
+
+# Add additional test suite definitions to the default test task here
+namespace :test do
+  desc 'Runs RuboCop on specified directories'
+  RuboCop::RakeTask.new(:rubocop) do |task|
+    # Dirs: app, lib, test
+    task.patterns = ['app/**/*.rb', 'lib/**/*.rb', 'test/**/*.rb']
```
Comment (tatwell) [inline lib/tasks/test.rake]: Here are files that will be rubocop'd when running tests.

**Comment by tatwell** [comment]
👍  Just a couple coding convention comments.

File: app/views/widgets/ad_landing_page/_masthead.html (hunk 1 above)
Comment (homeofmatt) [inline app/views/widgets/ad_landing_page/_masthead.html]: This is the embedded form code. I can give a shot at formatting it

File: app/views/widgets/ad_landing_page/_messaging_with_image.html (hunk 2 above)
Comment (homeofmatt) [inline app/views/widgets/ad_landing_page/_messaging_with_image.html]: I took it straight from the design

Comment (tatwell) [inline app/views/widgets/ad_landing_page/_messaging_with_image.html]: Then I'm ok with keeping markup as is. Just add a comment noting that -- and include the source. That could be helpful to future devs.

**Pull request merged**